tests/test_helpers/test_base.py

```diff
@@ -33,18 +32,16 @@ def test_locate(self):
         """Test the locate function to locate modules."""
         cwd = os.getcwd()
         os.chdir(os.path.join(CUR_PATH, ".."))
-        sys.modules["gym_connection"] = locate("packages.connections.gym")
-        assert sys.modules['gym_connection'] is not None
-        sys.modules["gym_connection"] = locate("packages.connections.weather")
-        assert sys.modules['gym_connection'] is None
+        assert locate("packages.fetchai.connections.gym") is not None
+        assert locate("packages.fetchai.connections.weather") is None
```

Comment:
We don't have a weather connection... so I'd add a comment that we are testing absence of a connection to make it easier to read